### PR TITLE
adding admin view to check status of syncs

### PR DESCRIPTION
**Pull request by joelhanson**

### CODE REVIEW

File: talentmap_api/integrations/views.py [hunk 1]
```diff
@@ -158,3 +158,18 @@ def get_queryset(self):
...
+                           GenericViewSet):
+    '''
+    list:
+    Lists all data sync tasks
+    '''
+    permission_classes = (IsAuthenticatedOrReadOnly, isDjangoGroupMember('superuser'))
+    serializer_class = SynchronizationTaskSerializer
+
+    def get_queryset(self):
+        pk = self.kwargs.get("pk")
```
Comment (rtirserio) [inline talentmap_api/integrations/views.py]: `pk` isn't needed

File: talentmap_api/integrations/views.py [hunk 2]
```diff
@@ -158,3 +158,18 @@ def get_queryset(self):
...
         logger.info(f"User {self.request.user.id}:{self.request.user} updating sync job entry {instance}")
+
+
+class DataSyncTaskListView(mixins.ListModelMixin,
+                           GenericViewSet):
+    '''
+    list:
+    Lists all data sync tasks
+    '''
+    permission_classes = (IsAuthenticatedOrReadOnly, isDjangoGroupMember('superuser'))
```
Comment (rtirserio) [inline talentmap_api/integrations/views.py]: Any reason this is `IsAuthenticatedOrReadOnly` vs `IsAuthenticated`?

**Comment by mjoyce91** [comment]
Might be unrelated, but I'm trying to test this by first running a sync job, but when I run `PUT /api/v1/data_sync/run/{id}/`, I get the error `local variable 'sync_task' referenced before assignment`.

File: talentmap_api/integrations/views.py (hunk 1 above)
Comment (joelhanson) [reply talentmap_api/integrations/views.py]: @rtirserio I missed removing this line from the copy/paste. I'll update.

File: talentmap_api/integrations/views.py (hunk 2 above)
Comment (joelhanson) [reply talentmap_api/integrations/views.py]: @rtirserio Following convention of the file. All the views there are `IsAuthenticatedOrReadOnly`

**Comment by mjoyce91** [comment]
Looks good. One thing I'm noticing (since I'm using a valid, mock WSDL, but one that obviously doesn't have the methods we're expecting) is that the synchronization_task gets created even when the task fails. I think that's a good thing, but I wonder if we want to add an additional field to say "failed: true". Otherwise, we just get a result with `null`s like this:

```json
{
      "id": 1,
      "start_date_time": null,
      "end_date_time": null,
      "running": false,
      "talentmap_model": "language.Language",
      "priority": 0,
      "use_last_date_updated": false,
      "new_count": null,
      "updated_count": null,
      "job": 10
}
```

**Comment by mjoyce91** [comment]
Also take a look at merge conflicts

**Comment by mjoyce91** [comment]
@joelhanson did you see my comment? https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/133#issuecomment-522029786

**Comment by joelhanson** [comment]
@mjoyce91 Did we decide that that's something we want? That seems like a separate story to update the sync task model.

**Comment by mjoyce91** [comment]
@joelhanson that's fair, I can create an issue and we can prioritize.

File: talentmap_api/integrations/views.py (hunk 2 above)
Comment (mjoyce91) [reply talentmap_api/integrations/views.py]: @rtirserio anything we need to look at here or everything good?

**Pull request merged**